tests/contracts/nonfungible-token/contract.scilla

```diff
@@ -72,6 +72,19 @@ let isApprovedOrOwner =
         let isOwnerOrApproved =  orb isOwner isApproved in
         orb isOwnerOrApproved isApprovedForAll
 
+(* Error events *)
+let makeErrorEvent =
+    fun (location: String) =>
+    fun (errorCode: Uint32) =>
+        {_eventname: "Error"; raisedAt: location; code: errorCode }
+
+let changeBoolToString =
+    fun (flag: Bool) =>
```

Comment:
Do you think we should rename this to `bool_to_string` and add to `BoolUtils.ml` ?